<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStart</class>
 <widget class="QDialog" name="GameStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="EasyGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModerateGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Moderate Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HardGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExpertGame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Expert Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartCustomGame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Custom Game</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="RowSelector">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="value">
    <number>14</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ColumnSelector">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MineSelector">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mines</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
